[2024-08-25T07:33:54.771+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T07:33:54.839+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.get_name scheduled__2024-08-20T00:00:00+00:00 [queued]>
[2024-08-25T07:33:54.891+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.get_name scheduled__2024-08-20T00:00:00+00:00 [queued]>
[2024-08-25T07:33:54.894+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T07:33:55.043+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-08-20 00:00:00+00:00
[2024-08-25T07:33:55.066+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=848) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T07:33:55.085+0000] {standard_task_runner.py:72} INFO - Started process 856 to run task
[2024-08-25T07:33:55.085+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v2', 'get_name', 'scheduled__2024-08-20T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpr0b_r03a']
[2024-08-25T07:33:55.096+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask get_name
[2024-08-25T07:33:55.409+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v2.get_name scheduled__2024-08-20T00:00:00+00:00 [running]> on host ae952724060e
[2024-08-25T07:33:56.000+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v2' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-20T00:00:00+00:00'
[2024-08-25T07:33:56.011+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T07:33:56.034+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Bruce', 'last_name': 'Banner'}
[2024-08-25T07:33:56.290+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T07:33:56.292+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v2, task_id=get_name, run_id=scheduled__2024-08-20T00:00:00+00:00, execution_date=20240820T000000, start_date=20240825T073354, end_date=20240825T073356
[2024-08-25T07:33:56.521+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T07:33:56.643+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-25T07:33:56.740+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
